beast/plotting/plot_param_err.py

```diff
@@ -2,13 +2,14 @@
 import matplotlib
```

Comment:
Does not look like this import is needed.  If this is true, please remove.

Reply:
Done!